lib/components/form.js

```diff
@@ -1,13 +1,49 @@
 import React from 'react'
+import request from 'axios'
 import { useSheet } from '../jss'
+import { endpoints } from '../config'
 
 const styles = {
 }
 
 const Form = React.createClass({
+  getInitialState() {
+    return {
+      account: '',
+      repository: '',
+      data: {
+        title: '---',
+        fields: []
+      }
+    }
+  },
+  componentDidMount() {
+    let { account, repository } = this.props.params
+    let json = `${endpoints.rawgit}${account}/${repository}/master/issue.json`
+    this.setState({
+      account: account,
+      repository: repository
+    })
+    this.serverRequest = request.get(json).then( (res) => {
```

Comment:
@watilde what is `$` on the react tips? I guess the tips said you could use `$` when you already use jQuery in the project. Thus we can use any library for Ajax and then I use axios.